starknet_devnet/starknet_wrapper.py

```diff
@@ -547,6 +548,37 @@ async def load_messaging_contract_in_l1(
             self.starknet, network_url, contract_address, network_id
         )
 
+    async def postman_l1_to_l2(
+        self,
```

Comment:
Can this accept a `CallL1Handler` object? This would be analogous to other methods which accepts an external tx and then convert it to an internal counterpart (`InternalL1Handler.from_external`).

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/fbcf227239fcd2da27dfbde8d95b46ddd310e018